Lombiq.HelpfulLibraries.OrchardCore/ResourceManagement/ResourceFilterBuilder.cs

```diff
@@ -52,6 +52,17 @@ public ResourceFilter When(Func<HttpContext, Task<bool>> filterAsync)
     public ResourceFilter WhenPath(string path) =>
         When(context => context.Request.Path.Value?.Trim('/').EqualsOrdinalIgnoreCase(path.Trim('/')) == true);
 
+    /// <summary>
+    /// Adds a filter that matches any of the provided <paramref name="paths"/> to the list of
+    /// <see cref="ResourceFilters"/>.
+    /// </summary>
+    public ResourceFilter WhenPaths(params string[] paths)
+    {
+        var trimmedPaths = paths.Select(path => path.Trim('/'));
+        return When(context =>
+            trimmedPaths.Contains(context.Request.Path.Value?.Trim('/'), StringComparer.OrdinalIgnoreCase));
+    }
```

Comment:
`WhenPaths` is now working with only 1 parameter. So there is no difference between `WhenPath("test")` and `WhenPaths("test")`.  I would suggest modifying the original `WhenPath` function. And also modifying the description to reflect that it can handle multiple paths.